### PR TITLE
MOBILEAPPS-2027

### PR DESCRIPTION
**Pull request by aman-alfresco**
- fixed requires session crash

### CODE REVIEW

File: app/src/main/java/com/alfresco/content/app/activity/MainActivityViewModel.kt [hunk 1]
```diff
@@ -75,6 +76,11 @@ class MainActivityViewModel(
         if (session != null) {
             init(appContext, session)
         }
+
+        viewModelScope.on<ActionSessionInvalid> {
+            println("MainActivityViewModel. ActionSessionInvalid ${it.requiresReLogin}")
```
Comment (ankitgoyalgloballogic) [inline app/src/main/java/com/alfresco/content/app/activity/MainActivityViewModel.kt:81]: please remove print statments

File: data/src/main/kotlin/com/alfresco/content/data/AnalyticsManager.kt [hunk 2]
```diff
@@ -38,7 +58,7 @@ class AnalyticsManager(val session: Session = SessionManager.requireSession) {
      */
     fun taskFiltersEvent(name: String) {
         val params = repository.defaultParams()
-        val eventName = "event_${name.replace(" ","_")}"
+        val eventName = "event_${name.replace(" ", "_")}"
```
Comment (ankitgoyalgloballogic) [inline data/src/main/kotlin/com/alfresco/content/data/AnalyticsManager.kt:61]: are we changing the event name?

File: session/src/main/kotlin/com/alfresco/content/session/SessionManager.kt [hunk 3]
```diff
@@ -5,7 +5,10 @@ import com.alfresco.content.account.Account
 
 object SessionManager {
     var currentSession: Session? = null
-    val requireSession get() = requireNotNull(currentSession)
+//    val requireSession get() = requireNotNull(currentSession)
```
Comment (ankitgoyalgloballogic) [inline session/src/main/kotlin/com/alfresco/content/session/SessionManager.kt:8]: remove commented code

**Review by ankitgoyalgloballogic — CHANGES_REQUESTED** [review]

File: data/src/main/kotlin/com/alfresco/content/data/AnalyticsManager.kt (hunk 2 above)
Comment (aman-alfresco) [reply data/src/main/kotlin/com/alfresco/content/data/AnalyticsManager.kt:61]: we are not changing the event name. It just added the space over there on refactoring

File: session/src/main/kotlin/com/alfresco/content/session/SessionManager.kt (hunk 3 above)
Comment (aman-alfresco) [reply session/src/main/kotlin/com/alfresco/content/session/SessionManager.kt]: resolved

File: app/src/main/java/com/alfresco/content/app/activity/MainActivityViewModel.kt (hunk 1 above)
Comment (aman-alfresco) [reply app/src/main/java/com/alfresco/content/app/activity/MainActivityViewModel.kt]: done

**Review by ankitgoyalgloballogic — APPROVED** [review]

**Pull request merged**